lib/responses.test.js

```diff
@@ -410,4 +410,37 @@ describe(_.startCase(filename), function () {
       fn()(req, res);
     });
   });
+
+  describe('checkArchivedToPublish', function () {
+    const fn = lib[this.title];
+
+    it('throws an error if checkProps returns true', function () {
+      let req = {
```

Comment:
The `req` object can be declared as `const` as well since we are not reassigning it anywhere else
```suggestion
      const req = { ... },
        res = createMockRes
```